Source/santactl/Commands/SNTCommandStatus.m

```diff
@@ -190,6 +196,7 @@ - (void)runWithArguments:(NSArray *)arguments {
     printf("%s\n", [statsStr UTF8String]);
   } else {
     printf(">>> Daemon Info\n");
+    printf("  %-25s | %s\n", "Connected", driverConnected ? "Yes" : "No");
```

Comment:
Something like: "Driver Connected"?
Also, add this to the JSON output.

Reply:
Done